Content.Shared/Chemistry/Reagent/ReagentPrototype.cs

```diff
@@ -1,20 +1,19 @@
 using System.Collections.Frozen;
 using System.Linq;
+using Content.Shared.FixedPoint;
```

Comment:
not sure how much people care about dumb things like this but you can make this using statement back to where it was do avoid diff bloat